sunpy/coordinates/frames.py

```diff
@@ -350,6 +383,10 @@ class Helioprojective(SunPyBaseCoordinateFrame):
     >>> sc
     <SkyCoord (Helioprojective: obstime=2010-01-01T00:00:00.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate for 'earth'>): (Tx, Ty) in arcsec
         (0., 0.)>
+    >>> sc = SkyCoord(CartesianRepresentation(0*u.km, 0*u.km, 0*u.km), obstime="2010/01/01T00:00:00", frame="helioprojective")
```

Comment:
You really should pick some non-zero value for the "x" coordinate here, and non-zero values for the "y" and "z" coordinates would be good too.  It's important because the conversion from `CartesianRepresentation` to the typical HPC values will be non-intuitive to many (e.g., "x" is very different from "theta_x").